osu.Game.Tests/Gameplay/TestSceneStoryboardSamples.cs

```diff
@@ -176,5 +178,9 @@ public TestDrawableStoryboardSample(StoryboardSampleInfo sampleInfo)
             {
             }
         }
+
+        public AudioManager AudioManager => Audio;
+        public IResourceStore<byte[]> Files => null;
+        public IResourceStore<TextureUpload> CreateTextureLoaderStore(IResourceStore<byte[]> underlyingStore) => null;
```

Comment:
These implementations are all in pretty big classes, so I'd `#region` them, at least in order to be able to visually separate them out.